app/pages/multisig/MultiSignatureRouter.tsx

```diff
@@ -8,6 +8,13 @@ import MultiSignatureCreateProposal from './MultiSignatureCreateProposal';
 import SubmittedProposal from './SubmittedProposal';
 import ExportKeyView from './ExportKeyView/ExportKeyView';
 import CreateAccountTransactionView from './AccountTransactions/CreateAccountTransactionView';
+import AddBaker from './AccountTransactions/AddBaker';
```

Comment:
Yes, but since that view uses `transactionKind` from route parameters to identify which flow to show, and we have multiple flows pr. `transactionKind` for baker/delegation transactions, it's not feasible to work with.

Reply:
please do